test/e2e/storage/ubernetes_lite_volumes.go

```diff
@@ -92,12 +95,28 @@ func PodsUseStaticPVsOrFail(f *framework.Framework, podCount int, image string)
 		for _, config := range configs {
 			e2epod.DeletePodOrFail(c, ns, config.pod.Name)
 		}
-		for _, config := range configs {
-			e2epod.WaitTimeoutForPodNoLongerRunningOrNotFoundInNamespace(c, config.pod.Name, ns)
-			e2epv.PVPVCCleanup(c, ns, config.pv, config.pvc)
-			err = e2epv.DeletePVSource(config.pvSource)
-			framework.ExpectNoError(err)
+		var wg sync.WaitGroup
+		wg.Add(len(configs))
+		errChan := make(chan error)
+		for i := range configs {
+			go func(config *staticPVTestConfig) {
+				defer wg.Done()
+				e2epod.WaitForPodToDisappear(c, ns, config.pod.Name, labels.Everything(), framework.Poll, f.Timeouts.PodDelete)
+				e2epv.PVPVCCleanup(c, ns, config.pv, config.pvc)
+				if err = e2epv.DeletePVSource(config.pvSource); err != nil {
+					errChan <- err
```

Comment:
with `make(chan error)`, this will IMO block until there is a reader.

Reply:
Yeah. Can you just `ExpectNoError` here? I think that should be threadsafe?